cli/ballerina-cli/src/test/java/io/ballerina/cli/cmd/BuildCommandTest.java

```diff
@@ -917,15 +917,13 @@ public void testBuildWithCorruptedDependenciesToml() throws IOException {
         BuildCommand buildCommand = new BuildCommand(projectPath, printStream, printStream, false);
         new CommandLine(buildCommand);
         buildCommand.execute();
-        String buildLog = readOutput(true);
-        Assert.assertEquals(
-                buildLog.replaceAll("\r", ""),
-                getOutput("corrupted-dependencies-toml.txt").replaceAll("\r", ""));
+        String buildLog = readOutput(true).replace("\r", "").replace("\\", "/");
```

Comment:
We maintain separate test output files for Windows and Unix. We can simply fix the corresponding file here without changing the code. The file related to this test is https://github.com/ballerina-platform/ballerina-lang/blob/master/cli/ballerina-cli/src/test/resources/test-resources/command-outputs/windows/corrupted-dependencies-toml.txt

Reply:
Fixed in [842b3ee](https://github.com/ballerina-platform/ballerina-lang/pull/42990/commits/842b3ee056646b88a73ff9b672185f3c868e8e55)